src/components/cards/TradeCard/TradeRoute.vue

```diff
@@ -433,7 +433,7 @@ export default defineComponent({
       const prefix = prefixMap[chainId] || '';
       return props.sorReturn.isV1swap
         ? `https://${prefix}pools.balancer.exchange/#/pool/${id}`
-        : `https://${prefix}app.balancer.fi/#pool/${id}`;
+        : `https://${prefix}balancer.fi/#pool/${id}`;
```

Comment:
I think if we remove the `app.` here it will break for mainnet, so you will need to update the `prefixMap` [just above](https://github.com/balancer-labs/frontend-v2/blob/7a432cb0d06bdb9526e6b949ebf7745382e6ba3d/src/components/cards/TradeCard/TradeRoute.vue#L430). So that mainnet returns `app.`

Reply:
Added a separate check for v1 mainnet. Since adding `app.` to the prefix would break v1 url